features/wallet/impl/src/main/java/com/tangem/feature/wallet/presentation/wallet/state/factory/WalletStateFactory.kt

```diff
@@ -102,17 +99,32 @@ internal class WalletStateFactory(
         }
     }
 
-    fun getStateWithOpenBottomSheet(content: WalletBottomSheetConfig.BottomSheetContentConfig): WalletStateHolder {
+    fun getStateWithOpenBottomSheet(
+        content: WalletBottomSheetConfig.BottomSheetContentConfig,
+        onDismissRequest: () -> Unit,
+    ): WalletStateHolder {
+        return currentStateProvider().copySealed(
+            bottomSheet = WalletBottomSheetConfig(
+                isShow = true,
+                onDismissRequest = onDismissRequest,
+                content = content,
+            ),
+        )
+    }
+
+    fun getStateWithActionBottomSheet(tokenId: String, onDismissRequest: () -> Unit): WalletStateHolder {
```

Comment:
То же про коллбек, плюс
```suggestion
    fun getStateWithTokenActionsBottomSheet(tokenId: String, onDismissRequest: () -> Unit): WalletStateHolder {
```